IB_iterators.py

```diff
@@ -0,0 +1,77 @@
+import random
+
+# Task 1
+
+
+def gen_fa(path):
+    file = open(path)
+    lines = file.readlines()
+    print(lines)
+    file.close()
+    i = 0
+    while i < len(lines)-1:
+        yield lines[i].strip('\n') + ' ' + lines[i+1].strip('\n')
+        i = i + 2
```

Comment:
Не забываем про возможность выставить шаг в `range`
```suggestion
for i in range(0, len(lines) -1, 2)
        yield lines[i].strip('\n') + ' ' + lines[i+1].strip('\n')
```
Выглядит стильно, модно, молодёжно, но, к сожалению, будет работать только с последовательностями, записанными в 1 строку. В фаста файлах же обычно последовательности разделяются на несколько строк по 60 символов